sdk/src/main/java/com/horizen/account/api/rpc/types/TransactionArgs.java

```diff
@@ -0,0 +1,160 @@
+package com.horizen.account.api.rpc.types;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.horizen.account.api.rpc.handler.RpcException;
+import com.horizen.account.api.rpc.utils.RpcCode;
+import com.horizen.account.api.rpc.utils.RpcError;
+import com.horizen.account.proposition.AddressProposition;
+import com.horizen.account.state.Message;
+import com.horizen.account.transaction.EthereumTransaction;
+import com.horizen.account.utils.BigIntegerUtil;
+import com.horizen.evm.utils.Address;
+import org.web3j.crypto.Sign;
+import org.web3j.utils.Numeric;
+
+import java.math.BigInteger;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TransactionArgs {
+    public BigInteger type;
+    public Address from;
+    public Address to;
+    public BigInteger gas;
+    public BigInteger gasPrice;
+    public BigInteger maxFeePerGas;
+    public BigInteger maxPriorityFeePerGas;
+    public BigInteger value;
+    public BigInteger nonce;
+
+    // We accept "data" and "input" for backwards-compatibility reasons.
+    // "input" is the newer name and should be preferred by clients.
+    // Issue detail: https://github.com/ethereum/go-ethereum/issues/15628
+    public String data;
+    public String input;
+
+    // Introduced by AccessListTxType transaction.
+//    public AccessList[] accessList;
```

Comment:
it seems better to accept `accessList` and return an error, rather than just skip this field.
Need UT for this case